orchestrator-salt/src/main/resources/salt/salt/idbroker/init.sls

```diff
@@ -0,0 +1,38 @@
+{%- from 'idbroker/settings.sls' import idbroker with context %}
+{%- from 'gateway/settings.sls' import gateway with context %}
+{{ idbroker.knox_data_root }}/security/keystores/signkey.pem:
+  file.managed:
+    - contents_pillar: gateway:signkey
```

Comment:
These keys should be the same as the gateway or it would be better if it has its own?

Reply:
I don't know.

Reply:
IDBroker signing keys should ideally be different from Gateway signing material. Reason being that SSH to the gateway nodes is more available to users than to the IDBroker nodes. Compromise the signing keys and you could potentially create a JWT token from which you could acquire cloud access keys.